src/Commands/AdminCommand.php

```diff
@@ -131,6 +132,13 @@ protected function getUser($create = false)
             if (!$email) {
                 $email = $this->ask('Enter the admin email');
             }
+            
+            // check if user with given email exists
+            $User = $model::where('email', $email)->first();
```

Comment:
Please use lowercase (camel-case, though it doesn't matter here) for variable names.

Also, can't you use `->exists()` on the model instead since you're not using the object anywhere.  This will allow you to combine this line with the conditional below it

Reply:
True. Updated it.